Client/Frontend/Browser/Tabs/GridTabViewController.swift

```diff
@@ -248,34 +247,13 @@ class GridTabViewController: UIViewController, TabTrayViewDelegate, Themeable {
     /// The main interface for scrolling to an item, whether that is a group or an individual tab
     ///
     /// This method checks for the existence of a tab to focus on other than the selected tab,
-    /// and then, focuses on that tab. The byproduct is that if the tab is in a group, the
-    /// user would then be looking at the group. Generally, if focusing on a group and
-    /// NOT the selected tab, it is advised to pass in the first tab of that group as
-    /// the `tabToFocus` in the initializer
+    /// and then, focuses on that tab.
```

Comment:
```suggestion
    /// and then focuses on that tab.
```

Reply:
History highlights are on for everyone in production, I have no idea if that is a feature users like and engage with but removing it would be a much bigger production conversation and should probably be driven by product. 
Tab groups are off everywhere except beta and have been for over a year, so 🔥

Reply:
LGTM I just wanted to confirm (similar to @adudenamedruby's question) that this PR doesn't touch tabGroups on HistoryPanel?

Reply:
It removes tab groups, but history panel and the history section on the home page should be untouched.